### PR TITLE
install cosign

### PR DESCRIPTION
**Pull request by cpanato**
install cosign

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,6 +30,17 @@ RUN  \
 	rm $GORELEASER_DOWNLOAD_FILE && \
 	goreleaser -v
 
+# install cosign
+ARG COSIGN_VERSION=1.2.1
```
Comment (developer-guy) [inline Dockerfile:34]: what about using something like this?
https://docs.docker.com/develop/develop-images/multistage-build/#use-an-external-image-as-a-stage

File: Dockerfile [hunk 2]
```diff
@@ -30,6 +30,17 @@ RUN  \
...
+ARG COSIGN_SHA=490cb1941aa317cd24a0bd9f2fe38932805dbaaba0ae89c12ec8138d15bdd8a0
+RUN  \
+	COSIGN_DOWNLOAD_FILE=cosign-linux-amd64 && \
+	COSIGN_DOWNLOAD_URL=https://github.com/sigstore/cosign/releases/download/v${COSIGN_VERSION}/${COSIGN_SHA} && \
```
Comment (developer-guy) [inline Dockerfile:38]: I think there is no release file named like this, right?

Comment (cpanato) [reply Dockerfile:38]: no no i messed up :(

**Comment by cpanato** [comment]
@developer-guy thanks for the review and feedback I will update this during this week

File: Dockerfile (hunk 1 above)
Comment (developer-guy) [reply Dockerfile:34]: copy --from=gcr.io/projectsigstore/cosign:v1.2.1  /bin/cosign /usr/local/bin/cosign

https://github.com/philips-labs/spiffe-vault/blob/465654d738696af5585175d185dcf8e4399c2432/example/spiffe-vault-cosign/Dockerfile#L1

cc: @cpanato

Comment (cpanato) [reply Dockerfile]: done

**Comment by developer-guy** [comment]
> @developer-guy thanks for the review and feedback I will update this during this week

Thank you so much @cpanato, such a honor for me to hear things like this from you 🤝❤️ looks great 👍

**Review by developer-guy — APPROVED** [review]

**Pull request merged**